src/components/Error/TodoError.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import { useAppContextContainer } from '../../context/AppContext';
+
+type Props = {
+  error: string | null;
+};
+
+const TodoError = ({ error }: Props) => {
+  const { handleClickCloseError } = useAppContextContainer();
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
```

Comment:
You don't have to write those strings separated by coma. You can just write all those strings in single one.